crates/proof-of-sql/src/sql/parse/query_context_builder.rs

```diff
@@ -36,15 +38,16 @@ impl<'a> QueryContextBuilder<'a> {
     pub fn visit_table_expr(
         mut self,
         table_expr: &[Box<TableExpression>],
-        default_schema: Identifier,
+        default_schema: Ident,
```

Comment:
Since this is immediately getting converted, let's leave this as Identifier, so we don't have to use `expect`.

Instead, do the conversion inside QueryExpr::try_new and return an Err instead of panicking.

```suggestion
        default_schema: Identifier,
```